docs/source/faq.md

```diff
@@ -38,7 +38,7 @@ The UCX framework is maintained and supported by hardware vendors in addition to
 The framework design, data structures, and components are design to provide highly optimized access to the network hardware. 
```

Comment:
"designed" and the earlier use of "design" is discordant.  I'd just remove the first use of "design".

"highly optimized" sounds a bit over-the-top, but if you want to plug UCX's abilities, OK :)